src/cursor/cur_backup.c

```diff
@@ -89,7 +106,13 @@ __curbackup_close(WT_CURSOR *cursor)
 	 * discarded when the cursor is closed), because that cursor will never
 	 * not be responsible for cleanup.
 	 */
-	if (F_ISSET(cb, WT_CURBACKUP_LOCKER))
+	if (F_ISSET(cb, WT_CURBACKUP_DUP)) {
+		__backup_free(session, cb);
+		/* Make sure the original backup cursor is still open. */
+		WT_ASSERT(session, F_ISSET(session, WT_SESSION_BACKUP_CURSOR));
```

Comment:
Can there be more than one dup cursor?  If so, then a single session flag for backup dup is not going to work.

Reply:
I don't think so. For now I'll forbid it and we can support it later if necessary.